settings.ui

```diff
@@ -205,6 +205,39 @@
          </layout>
         </widget>
        </item>
+       <item>
+        <widget class="QGroupBox" name="groupBox_11">
```

Comment:
Would be nice if we could name these widgets as well (even if it's just a context prefix). For example, this one could be `eepromGroupBox`.

Reply:
I agree, I'll fix it for the EEPROM settings here and maybe fix the others in another PR?

Reply:
Agreed. Whether you updated them here or in a future pr, we should create an issue for this to help with visibility.

Reply:
I changed the EEPROM object names based on what we'd talk about on discord. I'll probably mention that in another issue/PR, assuming this naming convention is fine anyways.